src/ghost.js

```diff
@@ -67,8 +69,26 @@ class GhostThread extends EventEmitter {
    *
    * @return    {Array<Object>}      users online
    */
-  async getPosts () {
-    return [...this._backlog].map(msg => JSON.parse(msg)).sort((p1, p2) => p1.timestamp - p2.timestamp)
+  async getPosts (num = null) {
```

Comment:
Rather than having num passed as a parameter here I think we should put it as an optional parameter in the constructor and make it global. Something like `this._backlogLimit`. The main reason we want this is to only send this number of messages to other peers when they join.

Reply:
Use 50 as default.

For example you could call `space.joinThread('name', { ghost: true, backlogLimit: 123 })`